### DISCUSSION TITLE
Outline Effect/Border Effect

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#644] [Ideas]
Allow to make a border outline over an animation layer or just a drawing layer. One of the style of do for my animation has a white outline around my character but I can not do that in ToonSquid so I have to transfer my animation to Clip Studio to add the white outline and then put it back to ToonSquid. It would be cool if outline effects can be added to animation layers or just a drawing layer without leaving ToonSquid allowing it to work for both vector and raster. Thanks.

**Comment by Lioncub901** [comment]
I don't know if this is appropriate but can this be added in the 2.0 update. We have already wait a long time it would not hurt to wait a bit more.

**Reply by Lioncub901** [reply]
also add way to add border on top of border (and more)

**Comment by DWrightAnimation** [comment]
Is this Outline effect your asking for the same one they'd use in shows like The Fairly OddParents or The Grim Adventures of Billy and Mandy, where it's basically a thin-line style with an extra outline around it to make it look thick-lined?

**Comment by RockmanEX3** [comment]
The closest thing to what we currently have is Drop Shadow, where the opacity is 100%.  Though it’s not really a complete outline.

**Reply by Lioncub901** [reply]
True

**Comment by Kosdaos1** [comment]
+1

**Comment by megamilkybee** [comment]
This would be amazing and allow for making game assets so much easier

**Comment by Miokaya** [comment]
Yes please

**Comment by megamilkybee** [comment]
Visual example of outline effect, it would be amazing to have something that works together with clipping masks to allow quick creation of clean art

https://github.com/user-attachments/assets/673506ef-c43e-4c99-9640-fe1d0cf0bf48


![IMG_7883](https://github.com/user-attachments/assets/4d88ac2a-d9c6-47f0-927e-e45fd8122255)
